src/Google.Storage.V1/StorageClientImpl.ListBuckets.cs

```diff
@@ -25,29 +25,30 @@ namespace Google.Storage.V1
     // ListBuckets methods on StorageClient
     public sealed partial class StorageClientImpl : StorageClient
     {
-        private static readonly PageStreamer<Bucket, BucketsResource.ListRequest, Buckets, string> s_bucketPageStreamer =
-            new PageStreamer<Bucket, BucketsResource.ListRequest, Buckets, string>(
-                (request, token) => request.PageToken = token,
-                buckets => buckets.NextPageToken,
-                buckets => buckets.Items);
+        private sealed class BucketPageManager : IPageManager<BucketsResource.ListRequest, Buckets, Bucket>
+        {
+            internal static readonly BucketPageManager Instance = new BucketPageManager();
```

Comment:
Why is this internal, whereas below are public?
It's a private class, so shouldn't all be consistently public?